wandb/util.py

```diff
@@ -613,6 +613,8 @@ def json_friendly(  # noqa: C901
             obj = obj.flatten()[0]
         elif obj.size <= 32:
             obj = obj.tolist()
+        # confusingly at this point if obj.size > 32 obj is a numpy array and
+        # converted is true, changing converted to False broke tests
```

Comment:
yikes

Reply:
Like [6 tests](https://app.circleci.com/pipelines/github/wandb/wandb/25057/workflows/67979df3-ce65-4c3b-956a-684b6f2271e2/jobs/694866/tests) in `unit_tests.test_utils`

Reply:
Thanks! Will ticket this!